include/s6a/s6a_fd.h

```diff
@@ -41,8 +41,6 @@
 /*SMS-Only-Indication:             1-------*/
 #define ULR_FLAG_SMS_ONLY_IND (1<<7)
 
-#define EXT_MAX_REQ_BW_DL_AVP_CODE 554
```

Comment:
Why this is removed ?

Reply:
Previously, we didn't have this AVP support in freediameter. So, this was required.
Now that we have updated the dictionary, we can fetch the AVP code directly from the dictionary.